### PR TITLE
Our Services Page Done

### PR DESCRIPTION
**Pull request by HenryChen666**
Our Services (list procedures and prices) page should be done. We can look over it together next week.

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -19,6 +19,24 @@
...
+    <button mat-menu-item routerLink="/our-services">
+      <mat-icon>mood</mat-icon>
+      <span>Cosmetic Dentistry</span>
+    </button>
+    <button mat-menu-item routerLink="/our-services">
+      <mat-icon>mood</mat-icon>
+      <span>Restorative Dentistry</span>
+    </button>
+  </mat-menu>
+-->
```
Comment (seofernando25) [inline src/app/app.component.html:38]: Try to minimize commiting commented code

File: src/app/app.component.html [hunk 2]
```diff
@@ -34,6 +52,13 @@
       </button>
     </div>
 
+    <div fxShow="true" fxHide.gt-sm="true">
+      <button mat-icon-button (click)="sidenav.toggle()">
+        <mat-icon>Services</mat-icon>
```
Comment (seofernando25) [inline src/app/app.component.html]: Is this intended? It looks like this to me

image

image

**Review by seofernando25 — CHANGES_REQUESTED** [review]
The page looks pretty good! 

~~Maybe you can narrow down the unused css 
or perhaps move some of what you did use to the main stylesheet
eg: You have some css for `fa-facebook` and `fa-twitter` but you didn't use it~~

File: src/app/app.component.html (hunk 1 above)
Comment (HenryChen666) [reply src/app/app.component.html:38]: I have this part here so we can discuss during the meeting if we should have the tab under "About Us" or a new separate tab

Comment (seofernando25) [reply src/app/app.component.html:38]: Ah ok sorry 😅

File: src/app/app.component.html (hunk 2 above)
Comment (HenryChen666) [reply src/app/app.component.html]: Fixed

**Review by seofernando25 — APPROVED** [review]
Looks good to me 👍

**Comment by seofernando25** [comment]
You can merge it when you please BTW

**Pull request merged**